packages/material-ui/package.json

```diff
@@ -39,7 +39,11 @@
     "typescript": "tslint -p tsconfig.json \"{src,test}/**/*.{spec,d}.{ts,tsx}\" && tsc -p tsconfig.json && tsc -p tsconfig.build.json"
   },
   "peerDependencies": {
+    "@emotion/core": "^10.0.27",
+    "@emotion/styled": "^10.0.27",
     "@types/react": "^16.8.6",
+    "emotion": "^10.0.27",
```

Comment:
Emotion 11 requires using the `<ClassNames />` component - https://emotion.sh/docs/class-names for accessing `css` & `cx`, and I was trying to make everything work without the necessity of having `<ThemeProvider />`. We could wrap each of our component with it tough, and that would be ok as it would be used only in the styled version. Should we do that? The styled component would look something like:

```jsx
<ClassNames>
  ({ cx, css }) => (
     <SliderBase
       ...
      />
  )
</ClassNames>
```

I wasn't sure if this would impact perf, because we would need it on each component, but I am open for other ideas as well..

Reply:
Is it possible to handle this with `@emotion/core` exclusively? I imagine we could wrap it it and provide the default theme when not available (without using any theme provider).

Reply:
We decided to move to using `styled` exclusively, so we don't even need this anymore 👍